frontend/src/metabase-lib/filter.ts

```diff
@@ -71,7 +72,7 @@ export function defaultFilterOperatorName(
   stageIndex: number,
   column: ColumnMetadata,
 ): FilterOperatorName {
-  return "=";
+  return isTime(column) ? "<" : "=";
```

Comment:
The default operator for time columns is "Before" (`<`), so I had to extend `defaultFilterOperatorName`. Is that a sign we need to move this to CLJC?

Reply:
I think that this function should not exist; it's a component thing to select some operator as the default one.